libkbfs/root_metadata.go

```diff
@@ -843,23 +391,46 @@ func (md *RootMetadata) swapCachedBlockChanges() {
 	}
 }
 
-// TlfHandleExtensions returns a list of handle extensions associated with the TLf.
-func (md *RootMetadata) TlfHandleExtensions() (extensions []TlfHandleExtension) {
-	if md.ConflictInfo != nil {
-		extensions = append(extensions, *md.ConflictInfo)
-	}
-	if md.FinalizedInfo != nil {
-		extensions = append(extensions, *md.FinalizedInfo)
+// A ConstRootMetadata is a thin wrapper around a
```

Comment:
To me, "const" doesn't really imply that the object can't be stored. In fact it kind of implies the opposite.  I haven't read the rest of the code yet, but we should clarify here (maybe in a second paragraph) what kind of things the storage restriction is useful for.  Maybe find a better adjective than "Const" too -- "Temporary"?  "ConstEphemeral"?  "Unowned"?